glean_parser/lint.py

```diff
@@ -320,6 +320,32 @@ def check_empty_datareview(
         yield "List of data reviews should not contain empty strings or TODO markers."
 
 
+def check_potential_data_sensitivity_required(
+    metric: metrics.Metric, parser_config: Dict[str, Any]
+) -> LintGenerator:
+    # Only looking at event metrics
```

Comment:
If the lint's event-specific, then the name of the fn should say as much.

(( I foresee we'll want something for `object`, too, at some point (if not now) ))